frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-builder/PipelineBuilderForm.tsx

```diff
@@ -176,6 +176,7 @@ const PipelineBuilderForm: React.FC<PipelineBuilderFormProps> = (props) => {
                   values.formData.tasks.length === 0
             }
             resetLabel={t('pipelines-plugin~Cancel')}
+            shadow={false}
```

Comment:
shadow set to false because layout for pipelinebuilder is different from add flows and will require more changes. looking into it.